src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,51 @@
+.nav {
+  margin-right: 50px;
```

Comment:
so, what I mean is to remove this one too

Reply:
`+`